data/data-pipeline/data_pipeline/etl/sources/census/etl_utils.py

```diff
@@ -69,3 +72,35 @@ def get_state_information(data_path: Path) -> pd.DataFrame:
     df["fips"] = df["fips"].astype(str).apply(lambda x: x.zfill(2))
 
     return df
+
+
+def check_census_data(census_data_path: Path, census_data_source: str) -> None:
```

Comment:
Well, this is `census/etl_utils` so this file aggregates utilities used for the ETL processes related to the Census. I believe the `check_census_data` is going to be called from [this part of the code](https://github.com/usds/justice40-tool/blob/esfoobar-usds/764-data-source-s3/data/data-pipeline/data_pipeline/etl/sources/census/etl.py#L69) as well, and maybe others in the future, so I felt it's better to have it in this utils instead of in `etl_score_geo` only.

Reply:
makes sense!